internal/table/table_test.go

```diff
@@ -96,7 +98,7 @@ func TestTable_colWidths(t1 *testing.T) {
 				},
 				100,
 			},
-			[]int{23, 23, 27, 27},
+			[]int{22, 22, 27, 29},
```

Comment:
This is also a good example for how the previous `rescaleColumns` function adds less jitter.  Before your changes, the last two columns were equal length (as the had the same contents), now the last column is a little wider.  But that's probably just something that I am concerned about. :)

Reply:
Yeah totally get it :) But I think this only registers with small numbers like this. In practice this won't really affect the UX and keeps the code readability. I'm trying to weigh all sides, and since readability is already problematic here the math can suffer the compromise here.